### ISSUE TITLE
a11y: Color Contrast - Account and API Keys

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13705]
Insufficient color contrast for the borders of form control

## Actual Result

Some form control border have insufficient contrast with their background, making it difficult for people with low vision and color blindness to distinguish them.

## Instances

Foreground color: #DCDEE7
Background color: #FFFFFF
Contrast ratio: 1.34:1
Main Content 
- [ ] Filter input field

"Account and API Keys
Projects/Namespaces

Foreground color: #DCDEE7
Background color: #FFFFFF
Contrast ratio: 1.34:1
Main Content 
- [ ] Unselected checkboxes in table data (on default state) 

Inside background color: #FFFFFF
Outside background color: #F4F5FA
Contrast ratio: 1.08:1
- [ ] Checkboxes present in the header (unselected checkbox on default state) 
- [ ] Unselected checkboxes in table data (on hover state) 

"Foreground color: #3D98D3
Background color: #FFFFFF
Contrast ratio:  3.16:1
Main Content
- [ ] Back 
- [ ] https://vpat2.ui.rancher.space/v3 

Foreground color: #FFFFFF
Background color: #3D98D3
Contrast ratio: 3.16:1
Main Content
- [ ] Change Password
- [ ] Create API Key 
- [ ] Delete (available after checking any checkbox from table)
- [ ] View in API and Delete (available on activating more options button on hover state)

Foreground color: #BEC1D2
Background color: #FFFFFF
Contrast ratio: 1.78:1
Main Content 
- [ ] Filter 

Foreground color: #5D995D  
Background color: #FFFFFF
Contrast ratio: 3.39:1 
Main Content 
- [ ] Active

Foreground color: #FFFFFF
Background color: #F64747
Contrast ratio: 3.55:1
Main Content
- [ ] Expired

Foreground color: #5D995D
Background color: #E9EEF1
Contrast ratio: 2.90:1
Main Content
- [ ] Active (in selected state) 

Foreground color: #F64747
Background color: #FFFFFF
Contrast ratio: 3.55:1
Main Content
- [ ] Just now ago

## Expected Result

The contrast ratio between the foreground border color of the form field and background color should be sufficient.

**Comment by rak-phillip** [comment]
Fails the VPAT: No Notes. Original Comment:

> Some form control border have insufficient contrast with their background, making it difficult for people with low vision and color blindness to distinguish them.

Issue 91
